extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcProcessor.java

```diff
@@ -505,7 +506,7 @@ public ValidationPhaseBuildItem validate(ObserverRegistrationPhaseBuildItem obse
             List<ObserverConfiguratorBuildItem> observerConfigurationRegistry,
             List<UnremovableBeanBuildItem> unremovableBeans,
             BuildProducer<BytecodeTransformerBuildItem> bytecodeTransformer,
-            BuildProducer<SynthesisFinishedBuildItem> synthesisFinished) {
+            BuildProducer<SynthesisFinishedBuildItem> synthesisFinished, BeanArchiveIndexBuildItem index) {
```

Comment:
The `BeanArchiveIndexBuildItem` is no longer needed, or?

Reply:
Yes, good catch!